guides/slack.md

```diff
@@ -0,0 +1,67 @@
+ - Demand channels (prefixed `demand-`). These channels are used by groups focusing on how we engage with other organisations. Optional unless directly involved with this group.
+ - Group channels (such as `ops-` or `academy-`). Channels used by 
+ - Specific purpose channels (usually prefixed with their temporary purpose, for example `bid-`). Temporary channels used for teams fulfilling a specific related task. Optional unless involved with the task, and will be archived after completion.
+ - General channels (no prefix). Company wide chanels such as `announcements`, `help` or regional channels such as `manchester` and _community of practice_ channels prefixed with `cop-`, these channels are used for cross company communication and tend have channel specific usage expecations, which should be detailed in their channel topic.
+ - Off topic channels (prefixed `x-`). These channels are entirely optional to join and use, feel free to create off topic channels if your interest or hobby is not yet represented!
+ - Bot channels or 'firehose' channels. It is occasionally useful to setup bots that post information to specific channels periodically, it is advised that you only need to join these channels if you are involved in monitoring such information.
+ 
+## Mandatory channels
+
+As of writing, it is expected every slack user will be present in `#announcements`, this is so that everyone can be reached in case of an important announcement.
+There are no other channels mandatory for _all_ users, but it is expected you will be in channels relevant to your working group.
+
+## Private channels and group chats
+
+Often private channels will be set of for specific tasks and group chats may be used to have small private discussions if necessary. It's worth noting that everything posted on our slack is logged and could be subject to a Freedom of Information request (specifically for channels relating to public sector projects), so it is wise to always use slack as if your message could be read.
+ 
+# Notifications and Threads
+ 
+It's generally expected you'll be available in relevant, non-optional channels during working hours - this means project/group channels you're directly involved in and `#announcements`. When minimising notification noise, it's entirely reasonable for you to set availability with the teams you're working with. All other optional channels are exactly that, optional. It is also reasonable to mute slack entirely and check periodically, whatever works best for you, remembering that slack is the primary method of communication within the company while remote.
+
+## Threads
+
+We don't have a set etiquette regarding threads, though generally we try to keep specific discussions in general channels to threads. Some notable exceptions currently are `announcements`, and `help`, where threading posts is actively encouraged. Any channels with specifc threading (or other) requirements will describe this in the channel topic.
```

Comment:
Should we use this guide to set the etiquette be be more pro-thread now? Perhaps rephrase to suggest threads should be the default in all channels?

Reply:
Exactly the sort of discussion we should have! I'm in favour of threads, personally

Reply:
+1 as you've already mentioned minimising noise in the opening.